archetypes/data-playground.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
+draft: true
+image: img/data-playground/
+link: "add github link"
+description: 'Add Description Here'
+summary: " Add summary here"
+license: "Add License Here"
+producer_name: "Add Producer name Here"
+producer_link: "Add Producer Link"
+api_key: "Add Api key Here"
+api_type: "Add Api type Here"
+sdks: " Add sdk list seperate with comma"
+limits: "Add limits here"
+data: " Add data here"
+account_required: "Add required account"
```

Comment:
```suggestion
account_required: "Add Required Account Here"
```

Reply:
This should also be a `bool` - is the account required true/false